common/nymkkt/Cargo.toml

```diff
@@ -0,0 +1,47 @@
+[package]
+name = "nym-kkt"
+version = "0.1.0"
+authors = ["Georgio Nicolas <georgio@nymtech.net>"]
+edition = "2021"
+license.workspace = true
+
+[dependencies]
+arc-swap = { workspace = true }
```

Comment:
it seems that most of these dependencies, apart from `blake3` and `thiserror` are actually unused